addons/web/static/tests/legacy/views/fields/many2many_tags_avatar_field_tests.js

```diff
@@ -399,24 +399,25 @@ QUnit.module("Fields", (hooks) => {
         );
         // select first input
         await click(popover.querySelector(".o-autocomplete--dropdown-item"));
-        assert.strictEqual(popover.querySelectorAll(".o_tag").length, 3, "Should have 3 tags");
+        // first input is already selected so it should remain two tags
+        assert.strictEqual(popover.querySelectorAll(".o_tag").length, 2, "Should have 2 tags");
```

Comment:
You change the semantics of this test, I would rather click on the third (?) element (the one that was first before your change), to keep the exact same semantics.